### ISSUE TITLE
Div's on main page should have role=button tabindex=0 for screenreaders

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/student-dashboard-django#163]
Screenreaders won't see these as actionable without this role and tabindex since it's an onclick now

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

**Comment by jonespm** [comment]
Unfortunately screenreader users won't get as much value from this tool since it's based on visualizations, but they should be able to still navigate.

**Comment by gsilver** [comment]
@jonespm @zqian - works a treat. This is for keyboard users as well as screen reader users. Making it a button works for all.  A quibble - since it is a button it does not need tabindex="0" because that is true of all native controls - but I am quibblering.
